src/sentry/runner/commands/devservices.py

```diff
@@ -159,8 +159,11 @@ def exit_handler(*_):
 @click.option("--project", default="sentry")
 @click.option("--exclude", multiple=True, help="Service to ignore and not run. Repeatable option.")
 @click.option("--fast", is_flag=True, default=False, help="Never pull and reuse containers.")
+@click.option(
+    "--skip-only-if", is_flag=True, default=False, help="Skip 'only_if' checks for services"
```

Comment:
You don't need `default=False` for `is_flag=True`. Feel free to remove this in other occurences.

Reply:
I like that it's kind of explicit tho